dyninstAPI/h/BPatch_snippet.h

```diff
@@ -238,8 +238,10 @@ class BPATCH_DLL_EXPORT BPatch_constExpr : public BPatch_snippet {
     //  Creates a representation of a (signed long) value
     BPatch_constExpr(signed long value);
     
-    //  Creates a representation of an (unsigned long) value
```

Comment:
I generally agree with this, though it's probably worth a bit of thinking about whether we're dealing with a specified size (uint64_t) or a specified type (unsigned long long).

But cases where we're speaking about `sizeof(void*)` are cases that should almost always use Dyninst::Offset/Dyninst::Address (mutatee-side) or ptrdiff_t/size_t (mutator-side), and the portability typedef for the former should occur once and only once.